copado/main/default/classes/Project.cls

```diff
@@ -0,0 +1,32 @@
+public class Project extends DomainBuilder {
+
+    // CTORS
+
+    public Project() {
+        super(copado__Project__c.SObjectType);
+
+        name('Project');
+    }
+
+    /**

```

Comment:
No such comments please

Reply:
totally. Personal note while it was WIP (I did not clean up everything properly before opening this PR)